jdk-wrapper.sh

```diff
@@ -189,6 +189,10 @@ extension="tar.gz"
 if [ "${JDKW_PLATFORM}" = "macosx-x64" ]; then
   extension="dmg"
 fi
+# JDK 6 and before use .bin extension, note no specific macosx download available for these versions (using linux)
+if [ "${JDKW_VERSION:2:1}" -le "6" ]; then
+  extension="bin"
```

Comment:
Adding the `.bin` extensions here was so even MacOS users would be able to use JDK 6 and before versions. Nesting under Linux will nullify that.